com_utils/api_control.py

```diff
@@ -193,11 +193,36 @@ def search_total_popular_brand_name():
         search_popular_brand_name['category_name'] = api_data['data']['brand']['results'][0]['categoryName']
         search_popular_brand_name['api_1st_brand_name'] = brands[0]['keyword']
         search_popular_brand_name['api_30th_brand_name'] = brands[29]['keyword']
+
+        brand_names = []
+        for i in range(0, len(brands)):
+            brand_name = brands[i]['keyword']
+            brand_names.append(brand_name)
+            search_popular_brand_name['brand_names'] = brand_names
+
         return search_popular_brand_name
     else:
         print('베스트 PLP API 불러오기 실패')
 
 
+# 연관 브랜드가 1개인 브랜드명
+def search_brand_by_related_brand():
+    brand_list = search_total_popular_brand_name()['brand_names']
+
+    brand_name = ''
+    for brand_name in brand_list:
+        response = requests.get(
+            f'https://search-api.29cm.co.kr/api/v4/products/brand/keyword/?keyword={brand_name}')
+        if response.status_code == 200:
+            brand_data = response.json()
+            front_brand_count = len(brand_data['data'])
+            if front_brand_count == 1:
```

Comment:
```suggestion
            if len(brand_data['data']) == 1:
```
front_brand_count 변수를 다른곳에서 사용하는곳이 없어보여서 위에 줄 없애고 이렇게 써도 되겠네요.